contrib/test_jenkins.sh

```diff
@@ -537,6 +537,7 @@ build_gcc_latest() {
 			UCX_HANDLE_ERRORS=bt,freeze UCX_LOG_LEVEL_TRIGGER=ERROR $ucx_inst/bin/ucx_info -d
 			$MAKEP distclean
 			echo "ok 1 - build successful " >> build_gcc_latest.tap
+			module unload dev/gcc-latest
```

Comment:
thanks @hoopoepg 
hopefully when we break test_jenkins.sh to AZP stages each would run separately to not have leftovers..